### ISSUE TITLE
[3.0.0-rc1] Unable to load postPublish hook

### ISSUE THREAD

**Issue by RodolfoGS** [expo/sentry-expo#133]
### Summary of Issue 

I'm testing the `3.0.0-rc1` version and I have problems to make `expo publish`. I'm getting the following error:

```
Unable to load postPublish hook: 'sentry-expo/upload-sourcemaps'. The module does not export a function.
› Closing Expo server
› Stopping Metro bundler
Please fix your postPublish hook configuration
```

### Environment - output of `expo diagnostics` & the platform(s) you're targeting

```
  Expo CLI 3.22.3 environment info:
    System:
      OS: macOS 10.15.5
      Shell: 5.7.1 - /bin/zsh
    Binaries:
      Node: 14.4.0 - /usr/local/bin/node
      Yarn: 1.19.1 - /usr/local/bin/yarn
      npm: 6.14.5 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Android Studio: 4.0 AI-193.6911.18.40.6514223
      Xcode: 11.5/11E608c - /usr/bin/xcodebuild
    npmPackages:
      expo: ~38.0.1 => 38.0.8
      react: ~16.11.0 => 16.11.0
      react-dom: ~16.11.0 => 16.11.0
      react-native: ~0.62.2 => 0.62.2
      react-native-web: ~0.11.7 => 0.11.7
    npmGlobalPackages:
      expo-cli: 3.22.3
```

### Reproducible Demo

Repo: https://github.com/RodolfoGS/sentry-hook-issue
1. `git clone git@github.com:RodolfoGS/sentry-hook-issue.git`
2. `cd sentry-hook-issue`
3. `yarn install`
4. `expo publish`
5. You will get the error

### Steps to Reproduce

1. `expo init SentryHookIssue`
2. Select minimal Bare Workflow
3. `cd SentryHookIssue`
4. `expo install sentry-expo@3.0.0-rc1`
5. `yarn add @sentry/react-native`
6. `npx pod-install`
7. `yarn sentry-wizard -i reactNative -p ios android`
8. Update `App.js` to import `'sentry-expo'`
9. Update `app.json` to add the hook
10. `expo publish`
11. You will get the error

**Comment by cruzach** [comment]
thanks for pointing this one out! pushing a fix as we speak

**Comment by cruzach** [comment]
Okay, fixed in version `3.0.0-rc2`, install with `yarn add sentry-expo@next`
